src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -263,11 +262,6 @@ boolean hasColumns() {
 
   @Override
   Optional<ParquetChunkData> getSnapshot(final String filePath) {
-    // We insert the filename in the file itself as metadata so that streams can work on replicated
-    // mixed tables. For a more detailed discussion on the topic see SNOW-561447 and
-    // http://go/streams-on-replicated-mixed-tables
-    metadata.put(Constants.PRIMARY_FILE_ID_KEY, StreamingIngestUtils.getShortname(filePath));
```

Comment:
since we're removing this, lets also remove filePath from the method signature on the whole callstack starting in FlushService.distributeFlushTasks? AFAIK this was the only reason we were generating the blobPath in the code location pointed out in JIRA.